### ISSUE TITLE
Using keytab with opentsdb

### ISSUE THREAD

**Issue by leosouzadias** [OpenTSDB/asynchbase#148]
I'm trying to configure opentsdb in a secure Hbase, but I could only make it work by doing a "kinit" before starting opentsdb, and after the ticket expires, I need to renew it manually (using a scheduled kinit and restart opentsdb). 
Can't we use keytabs with Asynchbase to have a smoothier behavior?
Another issue is that I can only connect to standalone Hbase, I still can't connect to distributed HBASE, is it supported?

**Comment by manolama** [comment]
Yes, you can use a Keytab file to run with secured HBase (and distributed too). Setup a JAAS config file with contents similar to:
```
Client {
  com.sun.security.auth.module.Krb5LoginModule required
  useKeyTab=true
  useTicketCache=false
  keyTab="<PATH_TO_KEYTAB>"
  principal="<PRINCIPAL_NAME>";
};
```

And when starting the TSD, pass in the following JVM flags (you can edit the tsdb shell script)
```
-Dhbase.kerberos.regionserver.principal=hbase/<PRINCIPAL_NAME> 
-Dhbase.sasl.clientconfig=Client 
-Djava.security.auth.login.config=<PATH_TO_JAAS_CONF> 
-Dhbase.security.authentication=kerberos 
-Dzookeeper.sasl.client=false
```

**Comment by IDerr** [comment]
I think you can close this @manolama

**Comment by manolama** [comment]
Thanks @IDerr
